app/scripts/views/message.js

```diff
@@ -440,9 +440,9 @@ define([
                 msg;
 
             if (val.length > 0) {
-                if (form.find('.textarea > span').length > 0) {
-                    form.find('.textarea > span').each(function() {
-                        $(this).after($(this).find('span').data('unicode')).remove();
+                if (form.find('.textarea > img').length > 0) {
```

Comment:
No one doesn't understand that you mean smiles, so 
``
$smiles = form.find('.textarea > img');
``